easybuild/easyblocks/m/mummer.py

```diff
@@ -63,6 +63,14 @@ def configure_step(self):
 
         self.cfg.update('buildopts', 'all')
 
+    def build_step(self):
+        """Custom build procedure for MUMmer."""
+
+        # make sure that $CXXFLAGS is being passed down
+        self.cfg.update('buildopts', r'CXXFLAGS="$CXXFLAGS \$(DEFINES)"')
```

Comment:
I don't know what this `$(DEFINES)` does, but I think we need some additional step to make the MUMmer build use the right `CXXFLAGS`, because there are some hardcoded lines in its Makefile:
```
CXXFLAGS = -O3
CFLAGS = -O3
LDFLAGS  =
```
And there are additional Makefiles in subdirectories, but I think those inherit the values from this one in the root dir. So, we need to patch/remove/comment these lines. Should we do that here or in the easyconfig (@boegel)? Note that the new MUMmer version (4.0 beta) no longer uses this easyblock.

Reply:
The `$(DEFINES)` part is indeed not needed here, I guess that's copy-paste from somewhere else?
What does make sense is to also control `CFLAGS` in the same way. `$CC` and `$CXX` are already honored thanks to using `:=` in the `Makefile`.

The value used for `$CXXFLAGS` is properly inherited by the subdirectories already, see build log.

```suggestion
        # make sure that compiler options specified by EasyBuild are used
        self.cfg.update('buildopts', 'CXXFLAGS="$CXXFLAGS" CFLAGS="$CFLAGS"')

Reply:
Thanks, I wasn't sure if `$(DEFINES)` is needed or not, since it should contain some args related to compilation, but yes, I havent seen it anywhere either (except from one EC that I copied it from).

Reply:
Thought that I saw some `-O3` flags when I tested this yesterday, but I don't see them anymore, so maybe these were from `CFLAGS`? Looks okay now!